components/common/NotificationBox/NotificationBoxModal.vue

```diff
@@ -0,0 +1,285 @@
+        eventFilter.value.some((x) => x === item.interaction))
+  )
+}
+
+const getNotifications = () => {
+  const { collections: collectionData, events: allEventsData } =
+    useNotification($store.getters.getAuthAddress)
+  collections.value = collectionData as unknown as FilterOption[]
+  allEvents.value = allEventsData as unknown as Event[]
+}
+
+watch(allEvents, doSearch, {
+  immediate: true,
+})
+
+watch(collectionFilter, doSearch, { deep: true })
+watch(eventFilter, doSearch, { deep: true })
+
+watch(() => $store.getters.currentUrlPrefix, getNotifications, {
+  immediate: true,
+})
+const emit = defineEmits(['close'])
+</script>
+
+<style scoped lang="scss">
+@import '@/styles/abstracts/variables';
+.no-wrap {
+  white-space: nowrap;
+}
+.rounded {
+  border-radius: 2rem;
+  overflow: hidden;
+}
+.notification-box-modal {
+  .notification-modal-container {
+    position: fixed;
+    top: 0;
+    right: 0;
+    height: 100%;
+    padding-top: 83px;
+    max-width: 360px;
+    width: 100%;
+    @include mobile {
+      padding-top: 58px;
+      max-width: 100vw;
+    }
+    @include ktheme() {
+      background: theme('background-color');
+      border-left: 1px solid theme('border-color');
+    }
+    .modal-card-title {
+      @include ktheme() {
+        color: theme('text-color');
+      }
+    }
+    .modal-card-head {
+      background: unset;
+      padding: 1rem 2rem;
+      @include ktheme() {
+        border-bottom: 1px solid theme('border-color');
+      }
+    }
+    .modal-card-body {
+      padding: 0rem 0rem 1rem 0rem;
+      display: block;
+      @include ktheme() {
+        background-color: theme('background-color');
+      }
+    }
+    .notification-filter {
+      flex-shrink: 0;
+      flex-grow: 0;
+      position: sticky;
+      top: 0;
+      padding-right: 2rem;
+      padding-left: 2rem;
+      z-index: 1;
+      @include ktheme() {
+        background-color: theme('background-color');
+      }
+      & > div:last-child {
+        @include ktheme() {
+          border-bottom: 1px solid theme('k-grey');
+        }
+      }
+
+      .filter-item {
+        cursor: pointer;
+        margin-right: 0.25rem;
+        &:last-child {
+          margin-right: 0;
+        }
+      }
+      .filter-list {
+        .filter-item {
+          @include ktheme() {
+            border: 1px solid theme('k-shade');
+            &:hover {
+              border-color: theme('border-color');
+            }
+            &.activated {
+              color: theme('black');
+              background-color: theme('k-shade');
+            }
+          }
+        }
+      }
+    }
+    .empty-tip {
+      @include ktheme() {
+        color: theme('k-grey');
+        text-align: center;
+      }
+    }
+  }
+}
+</style>
+<style lang="scss">
+.notification-box-modal {
+  .modal-content {
+    border: none !important;
+    box-shadow: none !important;
+  }
+  .modal-background {
+    background-color: rgba(0, 0, 0, 0.17) !important;
```

Comment:
is it possible to use one of the theme colors?

Reply:
this one has a transparency value and the currently seems colors in theme do not include it. imo it's not necessary here to use it. wdyt?